articles/cognitive-services/Translator/quickstart-csharp-detect.md

```diff
@@ -190,6 +190,7 @@ static async Task Main(string[] args)
     await DetectTextRequest(subscriptionKey, host, route, breakSentenceText);
 }
 ```
+If you are using a Cognitive Services multi-service subscription, you must also include the `Ocp-Apim-Subscription-Region` in your request parameters. [Learn more about authenticating with the multi-service subscription](https://docs.microsoft.com/azure/cognitive-services/translator/reference/v3-0-reference#authentication). 
```

Comment:
This is good but not the only issue that customers are facing. They are unsure about the endpoints usage for both single/multi subscription

Reply:
This has to be added in all API support cases, not just this particular instance